src/Raven.Server/Documents/Patch/ConflictPatchRequest.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Jint.Native;
+
+namespace Raven.Server.Documents.Patch;
+
+/// <summary>
+/// Represents a <see cref="PatchRequest"/> that resolves a conflict between documents.
+/// </summary>
+public sealed class ConflictPatchRequest : ScriptRunnerCache.Key
+{
+    private readonly string _script;
+
+    /// <summary>
+    /// Functions that are forbidden when performing a conflict resolution script.
+    /// </summary>
+    private static readonly string[] ForbiddenFunctionsNames =
+    [
+        "put",
+        "del",
+        "load",
+        "loadPath",
```

Comment:
we have more methods that can load/modify data... e.g. `incrementCounter`
but I guess for starters that is fine

Reply:
Basically, that shouldn't really matter.
Anything that modify will error because we aren't running this in a write transaction.
So we may get not an awesome error, but good enough